src/named_tuple.cr

```diff
@@ -41,9 +41,9 @@ struct NamedTuple
   # Creates a named tuple from the given hash, with elements casted to the given types. See `#from`.
   #
   # ```
-  # NamedTuple(foo: String, bar: Int64).from({:foo => "world", :bar => 2})       # => {foo: "world", bar: 2}
-  # NamedTuple(foo: String, bar: Int64).from({"foo" => "world", "bar" => 2})     # => {foo: "world", bar: 2}
-  # NamedTuple(foo: String, bar: Int64).from({:foo => "world", :bar => 2}).class # => {foo: String, bar: Int64}
+  # NamedTuple(foo: String, bar: Int32).from({:foo => "world", :bar => 2})       # => {foo: "world", bar: 2}
+  # NamedTuple(foo: String, bar: Int32).from({"foo" => "world", "bar" => 2})     # => {foo: "world", bar: 2}
+  # NamedTuple(foo: String, bar: Int32).from({:foo => "world", :bar => 2}).class # => {foo: String, bar: Int32}
```

Comment:
Changing `Int64` to `Int32` for `bar` disables casting feature which this example is about.